src/util/Authentication.ts

```diff
@@ -18,13 +18,15 @@ export function getCookie(cName) {
 
     let jsonObj = null
     try {
+        if( result !== null){
         jsonObj = JSON.parse(result)
+        }
     } catch (error) {
         console.error('Invalid JSON string:', error)
     }
     return jsonObj;
 }
 
-export function deleteCookie(cName) {
-    setCookie(cName, null, null)
+export function deleteCookie(cName: string): void {
+    setCookie(cName,'', 1) 
```

Comment:
Can you change the second and third arguments here back to null. This way the cookie deletes instead of persisting as an empty value.